src/helpers/validateCamera.ts

```diff
@@ -0,0 +1,18 @@
+import { Camera } from "../models/Camera";
+import { Rover } from "../models/Rover"; 
+const isCameraValid = (camera: Camera, rover: Rover): boolean => {
+    const validCameras = (camera: Camera, rover: Rover): Camera[] => {
+    switch (rover) {

```

Comment:
instead of swithing / case-ing, how about we have an object that defines which rover has which cameras?

```
const roverCameras: Record<Rover, Camera[]> = {
  Curiosity: [...],
  Opportunity: [...],
  ...
}

function isCameraValid = (camera: Camera, rover: Rover): boolean => {
  return roverCameras[rover].includes(camera)
}

```